### ISSUE TITLE
Add releasing status to the README.md

### ISSUE THREAD

**Issue by Aisuko** [SkywardAI/chat-frontend#85]
Add 

```
[![Release Drafter 🚀](https://github.com/SkywardAI/chat-frontend/actions/workflows/release-drafter.yml/badge.svg)](https://github.com/SkywardAI/chat-frontend/actions/workflows/release-drafter.yml)
```

 to the README.md

**Comment by Aisuko** [comment]
Currently, we do not have status of release drafter on the README.md. So, we should add the code above to the README.md

**Comment by Aisuko** [comment]
Fixed by #86
